lib/game.rb

```diff
@@ -24,24 +24,25 @@ def set_state(state)
     def current_player
         return @players[@active_player_index]
     end
-    
-    def _random_player_index
-        return rand(@players.length)
-    end
 
     def play_a_position(position) 
         if (@state == GAME_STATE_STARTED)
             @board.add_mark(current_player().mark(), position)
             
-            _cycle_active_player()
+            cycle_active_player()
```

Comment:
You don't need parenthesis after the function call

Reply:
[method parens](https://github.com/rubocop-hq/ruby-style-guide#method-parens)
[method invocation parens](https://github.com/rubocop-hq/ruby-style-guide#method-invocation-parens)